ansible/README.md

```diff
@@ -0,0 +1,34 @@
+# ansible-microshift
+
+The purpose of this ansible automation is to help gain insight into the footprint and start times of MicroShift.
+At this stage we are capturing the first start as well as the second start (with cached images), as well as disk
```

Comment:
Should we replace "as well" by a comma in the sentence?

Reply:
Yeah that was a pretty bad sentence, let me know if it reads better now.